src/slash-bedrock/share/brl-fetch/distros/exherbo

```diff
@@ -0,0 +1,147 @@
+#!/bedrock/libexec/busybox sh
+#
+# Exherbo Linux bootstrap support
+# Copyright 2019 Wulf C. Krueger <philantrop@exherbo.org>
+# Distributed under the terms of the GNU General Public License v2
+#
+# based in part upon "Gentoo Linux bootstrap support" which is:
+#      This program is free software; you can redistribute it and/or
+#      modify it under the terms of the GNU General Public License
+#      version 2 as published by the Free Software Foundation.
+#
+#	Copyright (c) 2016-2019 Daniel Thau <danthau@bedrocklinux.org>
+
+. /bedrock/share/common-code
+trap 'fetch_abort "Unexpected error occurred."' EXIT
+
+check_supported() {
+	true
+}
+
+speed_test_url() {
+	echo "/stages/sha1sum"
+}
+
+list_mirrors() {
+	echo "https://dev.exherbo.org/"
+}
+
+brl_arch_to_distro() {
+	case "${1}" in
+	     "x86_64") echo "amd64" ;;
+	     *) abort "brl does not know how to translate arch \"${1}\" to ${distro:-} format" ;;
+	esac
+}
+
+list_architectures() {
+	cat <<EOF
+x86_64
+EOF
+}
+
+default_release() {
+	echo "rolling"
+}
+
+list_releases() {
+	echo "rolling"
+}
+
+fetch() {
+        paludis_builddir=/var/paludis
+        paludis_repo_dir=/etc/paludis/repositories
```

Comment:
paludis_repo_dir points to Exherbo's standard directory, yes. It's just there because I try to avoid verbatim strings among code. 

paludis_builddir is another beast: That's the shared /var/tmp thing we discussed on IRC as the standard directory would indeed be /var/tmp/paludis. 

Sharing /var/tmp could be problematic for similar reasons for at least Gentoo and Funtoo as well, AFAIK. So, making it local would be good, I'd say.

Reply:
Lets change this PR to make use the default `/var/tmp/` (or `/var/tmp/paludis`?) with the expectation that `/var/tmp` will be removed from the global list before this is merged.

Reply:
Yes, that would be best. I'll do that. And it's in fact /var/tmp/paludis. :-)